### PR TITLE
Readme rework

### PR DESCRIPTION
**Pull request by vancauwe**
Update of Readme towards something more comprehensive. 

Caveat: may be more complicated than what is stated in Windows.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -125,20 +107,49 @@ gh.serialize(format='ttl')
...
+Supported formats are turtle, json-ld and n-triples (by specifying the `--format` argument in your call i.e. `gimie data https://github.com/numpy/numpy --format 'ttl'`).
+
+With no specifications, Gimie will print results in the terminal. Want to save Gimie output to a file? Add your file path to the end : `gimie data https://github.com/numpy/numpy > path_to_output/gimie_output.ttl`
 
 ### Limitations
 
-* Currently, gimie will only the first 100 contributors of a repository (in arbitrary order), and for each users, at most 100 affiliations.
-* If a Github repository is owned by an organization, all "mentionable users" are reported as contributors. This will include all members of the organization in addition to contributors.
+* For a Github repository extraction: if the GitHub repository is owned by an organization, all "mentionable users" are reported as contributors. This will include all members of the organization in addition to contributors.
```
Comment (cmdoret) [inline README.md:117]: This limitation has fortunately been addressed by https://github.com/SDSC-ORD/gimie/pull/33#issuecomment-1476652928. Should we then drop the `Limitations` section?

**Review by cmdoret — COMMENTED** [review]
LGTM! I would just remove the outdated limitation, but this looks great :)

File: README.md (hunk 1 above)
Comment (vancauwe) [reply README.md:117]: yep sounds good

**Comment by vancauwe** [comment]
ready to merge I think

**Pull request merged**